fontforge/sfd.c

```diff
@@ -7349,9 +7349,11 @@ static OTLookup **SFDLookupList(FILE *sfd,SplineFont *sf) {
     }
     if ( lcnt==0 )
 return( NULL );
-
     ret = malloc((lcnt+1)*sizeof(OTLookup *));
     memcpy(ret,buf,lcnt*sizeof(OTLookup *));
+    // check if buf uses heap after a malloc/realloc
+    if (lcnt>100)
```

Comment:
Let's match the check above and do `if (buf != space)`

Reply:
I agree that that test makes more sense. But, why aren't we just dynamically allocating from the start? This hybrid model creates a lot of unnecessary complexity.

Reply:
@frank-trampe  I've updated it; take a look